roles/cobbler/templates/kickstarts/cephlab_rhel.ks

```diff
@@ -33,7 +33,7 @@ selinux --permissive
 # Do not configure the X Window System
 skipx
 # System timezone
-timezone  America/New_York
+timezone Etc/UTC --utc
```

Comment:
I see that; I started with the Ubuntu doc below that says "look at /usr/share/zoneinfo".  My bet is that the python script/list is lazily ignoring all the stuff at the top level, and I bet both work; they certainly both work for 

[ubuntu@smithi028 ~]$ TZ=UTC date
Wed Mar 16 01:08:30 UTC 2016
[ubuntu@smithi028 ~]$ TZ=Etc/UTC date
Wed Mar 16 01:08:36 UTC 2016

That said, we can leave it.